api/core/core.proto

```diff
@@ -183,6 +202,62 @@ message GetInventoryResponse {
     repeated InventoryEntry entries = 1;
 }
 
+message PolicyValidation{
+    string id = 1;
+    string message = 2;
+    string cluster_id = 3;
+    string category = 4;
+    string severity = 5;
+    string created_at = 6;
+    string entity=7;
+    string namespace=8;
+    string violating_entity=9;
+    string description=10;
+    string how_to_solve=11;
+    string name=12;
+    string clusterName = 13;
+    repeated PolicyValidationOccurrence occurrences = 14;
+    string policy_id = 15;
```

Comment:
Mixing camel-case with snake case here. Can we use camel-case for proto fields?

Reply:
done